src/averageAges.js

```diff
@@ -15,12 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+  const men = century
+    ? people.filter(person => century === Math.ceil(person.died / 100)
+      && person.sex === 'm')
+    : people.filter(person => person.sex === 'm');
```

Comment:
you repeat the filer logic
this might be better:
```suggestion
  const men = people.filter(person => (
    person.sex === 'm'
    && (century ? century === Math.ceil(person.died / 100) : true)
```

or even this 
```suggestion
  const men = people.filter(person => (
    person.sex === 'm'
    && (century && century === Math.ceil(person.died / 100))
```